New Group/HomeScreen/Cell/ListMovieTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+
+import UIKit
+
+class ListMovieTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var posterImage: UIImageView!
+    @IBOutlet weak var nameMovieLabel: UILabel!
+    @IBOutlet weak var averageRaitingLabel: UILabel!
+    @IBOutlet weak var releaseDateLabel: UILabel!
```

Comment:
You have the func fillData, so these properties should be private or fileprivate